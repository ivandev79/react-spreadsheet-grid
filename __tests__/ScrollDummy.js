import React from 'react';
import renderer from 'react-test-renderer';
import ScrollDummy from './../src/scrollDummy';

test('ScrollDummy with rows', () => {
    const tree = renderer.create(
        <ScrollDummy
            headerHeight={50}
            rowHeight={50}
            rows={[1, 2, 3]}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});

test('ScrollDummy without rows', () => {
    const tree = renderer.create(
        <ScrollDummy
            headerHeight={50}
            rowHeight={50}
            rows={[]}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});