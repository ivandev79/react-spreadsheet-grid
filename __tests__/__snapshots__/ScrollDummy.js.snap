// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollDummy with rows 1`] = `
<div
  className="SpreadsheetTableScrollDummy"
  style={
    Object {
      "height": "200px",
    }
  }
/>
`;

exports[`ScrollDummy without rows 1`] = `
<div
  className="SpreadsheetTableScrollDummy"
  style={
    Object {
      "height": "50px",
    }
  }
/>
`;
